examples/run_largecyc_generic_echo.py

```diff
@@ -0,0 +1,137 @@
+"""
+Spin Echo
+=========
+
+A Hahn echo is generated by feeding the generic ppg the specific ppg_list
+required. Note that the overall phase and the 90-180 phase difference are
+phase cycled in a nested way similar to run_CPMG.py. If you wish to keep
```

Comment:
This description of the phase cycling no longer applies... Give the phase argument here as well like I did in my pull request.